### ISSUE TITLE
Data missing for Azure AKS Deployment Tutorial

### ISSUE THREAD

**Issue by ChristopherEeles** [snakemake/snakemake#1370]
The __[](https://snakemake.readthedocs.io/en/stable/executor_tutorial/azure_aks.html#auto-scaling-azure-kubernetes-cluster-without-shared-filesystem)Auto-scaling Azure Kubernetes cluster without shared filesystem__ tutorial has a broken link when trying to download `snakedir.zip`, which contains the environment files needed to complete the tutorial.

The broken link is in the __Running the workflow__ section and is as follows:
https://github.com/snakemake/snakemake/blob/main/docs/executor_tutorial/workflow/snakedir.zip

Please upload this data or provide an alternative link. I would very much like to complete your very helpful tutorial.

Best,
Christopher Eeles
Software Developer
BHK Lab | PM-Reseach | UHN

**Comment by aryarm** [comment]
Yeah, this is definitely an issue and has been brought up before. See #1074 and #1315
Duplicate of #1074

I think the solution would be to put the files in their own git repo, as was done for the GLS tutorial. Or maybe it could just share the GLS tutorial repo?

**Comment by aryarm** [comment]
Duplicate of #1074
